### PR TITLE
[en]fix terms in Virtnest

### PR DESCRIPTION
**Pull request by ModetaNiu**
Related to #4956
and update MTPE
术语修改：
click on -> click
configuration item -> configmap
container group -> Pod
elastic scaling, expansion and contraction -> autoscaling 自动扩缩容
etc -> 大部分情况可删除
GPU cards -> GPUs
image repository, image repositories -> container registry 镜像仓库
https://docs.daocloud.io/... 非图片 -> 相对路径
image space -> registry space
micro-service -> microservice
multi-cloud -> multicloud
seed cluster, seed node -> bootstrap cluster, bootstrap node
stateful workload -> statefulset
stateless workload -> deployment
strategy -> policy
table 减少冗余的空格
Title 用 [Chicago Capitalization Style](https://capitalizemytitle.com/style/Chicago/)
Heading 仅第一个单词首字母大写即可
the load instance -> the workload instance
time window -> period
timed task, scheduled task, cron-task -> CronJob
version v0.1 -> v0.1
working cluster, work cluster -> worker cluster
⋮ -> ┇
fix anchors 中英文锚点不同
命令、代码用 ``；UI 菜单、按钮等用粗体
代码段的注释，建议[隐藏为 +](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-annotations)（带 | 的代码段不支持隐藏注释）

### CODE REVIEW

File: docs/en/docs/virtnest/import/import-ubuntu.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ into the virtual machines of DCE 5.0 through the command line.
 
     - Download vddk: Need to register an account on the [VMware website](https://developer.vmware.com/) to download
    
-        Go to SDKs, click on "_Compute Virtualization_" in the section, and select the appropriate version of
+        Go to SDKs, click "_Compute Virtualization_" in the section, and select the appropriate version of
```
Comment (windsonsea) [inline docs/en/docs/virtnest/import/import-ubuntu.md:65]:
```suggestion
        Go to SDKs, click __Compute Virtualization__ in the section, and select the appropriate version of
```
粗体、斜体、引号，有一个就行，不用 double

File: docs/en/docs/virtnest/vm/vm-gpu.md [hunk 2]
```diff
@@ -76,23 +76,23 @@ The following are the steps to build the vGPU Manager image and push it to the i
...
+7. Push the NVIDIA vGPU Manager image to your container registry
 
     ```bash
     docker push ${PRIVATE_REGISTRY}/vgpu-manager:${VERSION}-${OS_TAG}
     ```
 
 ## Label Cluster Nodes
 
-Go to **Container Management** , select your working cluster, click **Nodes** in the action bar, and click **Edit Labels** to add labels to the nodes. Each node can only have one label.
+Go to **Container Management** , select your worker cluster, click **Nodes** in the action bar, and click **Edit Labels** to add labels to the nodes. Each node can only have one label.
```
Comment (windsonsea) [inline docs/en/docs/virtnest/vm/vm-gpu.md:87]: 这句中文原句不太对：

```
进入 **容器管理** ，选取您的工作集群，在节点列表右侧，点击 **┇**，选择 **修改标签** ，为节点添加标签。每个节点只能有一种标签。
```

File: docs/en/docs/virtnest/vm/vm-gpu.md [hunk 3]
```diff
@@ -102,7 +102,7 @@ You can assign the following values to the labels: container, vm-passthrough, an
         
         // Set the following fields only if you need vGPU
         gpu-operator.vgpuManager.enabled=true
-        gpu-operator.vgpuManager.repository=<your-register-url>      // Image repository address in the "Build vGPU Manager Image" step
+        gpu-operator.vgpuManager.repository=<your-register-url>      // container registry address in the "Build vGPU Manager Image" step
```
Comment (windsonsea) [inline docs/en/docs/virtnest/vm/vm-gpu.md:105]:
```suggestion
        gpu-operator.vgpuManager.repository=<your-register-url>      // Container registry address in the "Build vGPU Manager Image" step
```
全局替换要区分大小写

File: docs/en/docs/virtnest/template/index.md [hunk 4]
```diff
@@ -36,27 +47,32 @@ On the Create VMs page, enter the information according to the table below and c
 
 The template list will appear, and you can choose either an internal template or a custom template based on your needs.
 
-- Select an Internal Template: The platform provides three standard templates that cannot be edited or deleted. When selecting an internal template, the image source, operating system, image address, and other information will be based on the template and cannot be modified. Resource quotas will also be based on the template but can be modified.
+- Select an Internal Template: The platform provides two standard templates that cannot be edited or deleted. When selecting an internal template, the image source, operating system, image address, and other information will be based on the template and cannot be modified. GPU quota will also be based on the template but can be modified.
```
Comment (windsonsea) [inline docs/en/docs/virtnest/template/index.md:50]:
```suggestion
- Select an Internal Template: DCE 5.0 Virtual Machine provides several standard templates that cannot be edited or deleted. When selecting an internal template, the image source, operating system, image address, and other information will be based on the template and cannot be modified. GPU quota will also be based on the template but can be modified.
```
中英文可以用模糊词：几个

**Comment by windsonsea** [comment]
https://docs.daocloud.io/virtnest/vm/vm-gpu/#vgpu-manager

这段代码不太对，去问问 gpt：

```
docker build \
  --build-arg DRIVER_VERSION=${VERSION} \
  --build-arg CUDA_VERSION=${CUDA_VERSION} \
  -t ${PRIVATE_REGISTRY}``/vgpu-manager``:${VERSION}-${OS_TAG} .
```

File: docs/en/docs/virtnest/vm/vm-gpu.md (hunk 2 above)
Comment (ModetaNiu) [reply docs/en/docs/virtnest/vm/vm-gpu.md:87]: See revision in #4983

**Review by windsonsea — APPROVED** [review]

**Pull request merged**